manifests/g/GIGABYTE/LaunchAppCenter/17.1204.1/GIGABYTE.LaunchAppCenter.installer.yaml

```
@@ -0,0 +1,16 @@
+# Created using wingetcreate 1.9.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+
+PackageIdentifier: GIGABYTE.LaunchAppCenter
+PackageVersion: 17.1204.1
+InstallerType: zip
+NestedInstallerType: portable
+NestedInstallerFiles:
+- RelativeFilePath: AppCenter\setup.exe
+  PortableCommandAlias: appcenter
```

```suggestion
NestedInstallerType: exe
NestedInstallerFiles:
- RelativeFilePath: AppCenter\setup.exe
InstallerSwitches:
  Silent: /silent
  SilentWithProgress: /silent
ElevationRequirement: elevatesSelf
```

I don't think this is correct, based on the installer I got, this should be an InstallShield installer.
[WinGet doesn't seem to support automatically defining silent install switches for this type of installer](https://github.com/microsoft/winget-pkgs/blob/master/doc/manifest/schema/1.10.0/installer.md) at this time, which needs to be explicitly defined here.
Based on the silent install switches on other GIGABYTE products, I'm guessing its silent install switch is `/silent`.